sphereon-kmp-crypto-kms-azure/src/jsMain/kotlin/com/sphereon/crypto/kms/azure/AzureKeyVaultMapper.js.kt

```diff
@@ -0,0 +1,45 @@
+package com.sphereon.crypto.kms.azure
+
+import com.sphereon.crypto.jose.JoseKeyOperations
+import com.sphereon.crypto.jose.JwaAlgorithm
+import com.sphereon.crypto.jose.JwaKeyType
+import com.sphereon.crypto.jose.Jwk
+
+private fun String.toJwaKeyType(): JwaKeyType {
+    return when (this) {
+        "EC" -> JwaKeyType.EC
+        else -> throw IllegalArgumentException("Unsupported key type: $this")
+    }
+}
+
+private fun String.toJoseKeyOperationsArray(): JoseKeyOperations {
+    return when (this) {
+        "sign" -> JoseKeyOperations.SIGN
+        "verify" -> JoseKeyOperations.VERIFY
+        "encrypt" -> JoseKeyOperations.ENCRYPT
+        "decrypt" -> JoseKeyOperations.DECRYPT
+        "wrapKey" -> JoseKeyOperations.WRAP_KEY
+        "unwrapKey" -> JoseKeyOperations.UNWRAP_KEY
+        else -> throw IllegalArgumentException("Unsupported key operation: $this")
+    }
+}
+
+fun String.toJwaAlgorithm(): JwaAlgorithm {
+    return when (this) {
+        "P-256" -> JwaAlgorithm.ES256
+        "P-384" -> JwaAlgorithm.ES384
+        "P-521" -> JwaAlgorithm.ES512
+        else -> throw IllegalArgumentException("Unsupported algorithm: $this")
+    }
+}
+
+fun AzureKeyvaultKey.toJwk(): Jwk {
+    return Jwk(
+        alg = key.crv.toJwaAlgorithm(),
+        kid = key.kid,
+        kty = key.kty.toJwaKeyType(),
+        key_ops = key.keyOps.map { it.toJoseKeyOperationsArray() }.toTypedArray(),
+        n = key.n.toString(),
```

Comment:
This looks like RSA JWKs. Where are the x and y pub key and potential d priv key? 

we can ignore the d value, as we do not want to expose the private key anyway, which in most cases would not be possible to begin with